test/indexer/test_hca_indexer.py

```diff
@@ -538,37 +545,69 @@ def test_contributor_matrices(self):
             ]
         }
 
+        project2_expected_contributor_matrices = {
+            'file': [
+                {
+                    # Analysis file. The 'strata' value was gathered by walking
+                    # the project graph. Source from file_source.
+                    'uuid': '2ecedea4-b90c-5025-9967-fc08f27a4dc6',
+                    'version': '2021-01-01T00:00:00.000000Z',
+                    'name': 'AP1_file.h5ad',
+                    'size': 143,
+                    'matrix_cell_count': 2100,
+                    'source': 'Contributor',
+                    'strata': 'genusSpecies=Homo sapiens;'
+                              'developmentStage=human adult stage;'
+                              'organ=blood;'
+                              'libraryConstructionApproach=10x v3 sequencing'
+                }
+            ]
+        }
+
         num_docs = Counter()
+        contents = []  # Contents from project aggregate hits
         for hit in hits:
             entity_type, aggregate = self._parse_index_name(hit)
             num_docs[entity_type, aggregate] += 1
             if entity_type == 'projects' and aggregate:
-                contents = hit['_source']['contents']
-                self.assertEqual(expected_matrices, one(contents['matrices']))
-                self.assertEqual(expected_contributor_matrices, one(contents['contributor_matrices']))
+                contents.append(hit['_source']['contents'])
```

Comment:
This seems to be sensitive to the ordering of the hits. That is brittle and confusing. Please key the expectation and the actual values on the project id. The all  of `project1...` and `project2...` stuff disappears.